### DISCUSSION TITLE
Welcome to cfast Discussions!

### DISCUSSION THREAD

**Discussion by rmcdermo** [firemodels/cfast#2056] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Antonella-unige** [comment]
Buongiorno, non riesco a far funzionare CFAST su un pc aziendale con sistema operativo Windows 11 Education. Sapete se CFAST è incompatibile con Windows 11 Education o potete aiutarmi in qualche modo? Grazie mille!

**Reply by mcgratta** [reply]
What is the error message? It is possible that Windows 11 Education edition might be missing a library needed by CFAST.

**Reply by Antonella-unige** [reply]
Avevo installato CFAST ed ero riuscita a disegnare tutta la geometria dell'edificio, compreso l'inserimento di apertura di ventilazione e del fuoco. Alla prima elaborazione dello scenario di incendio, CFAST si è arrestato in modo anomalo e da quel momento in poi non sono più riuscita né ad aprire il file sul quale avevo lavorato né a re-installare dall'inizio CFAST.
La finestra che compariva è questa:
![schermata di errore](https://github.com/user-attachments/assets/4bfe0c89-c5d6-452d-ab63-87901b5cc2a8)

Il testo della schermata è questo: . NET-BroadcastEventWindow.4.0.0.1ca0192.0: CEdit.exe - errore di eccezione "Si è verificato l'errore di exception eccezione software sconosciuta (0xe0434352) nell'applicazione alla posizione 0x0000000076D33874"

Pensi possa essere legato alla versione del sistema operativo o sia dovuto ad altro?

**Reply by mcgratta** [reply]
"Unknown software exception" can mean almost anything. Do the following:
1. Post your input file (the file with the ".in" suffix)
2. Try your case on another computer to see if the error is repeated.

**Reply by Antonella-unige** [reply]
Grazie mille per il tuo suggerimento, McGratta :)
Ho verificato che il file con estensione .in (lo stesso sul quale stavo lavorando) si apre regolarmente su un computer diverso dal mio. 
Ma io devo lavorare sul pc aziendale a me in uso, non su un altro. Il problema sta proprio in questo. 
Sul pc aziendale che uso non riesco più ad installare CFAST e, di conseguenza, ad aprire il file .in sul quale stavo lavorando. 
Si è ipotizzato fosse un problema legato al sistema operativo Windows 11 Education, presente sul pc aziendale a me in uso, ma io non ne sono troppo convinta perché, secondo me, non sarei dovuta riuscire già in partenza ad installare e usare CFAST.
Ti viene in mente un'altra possibile causa/soluzione al problema?

**Reply by mcgratta** [reply]
It is possible that the problem is with the graphical user interface for CFAST. The CFAST source code itself is just Fortran and can be executed at the command line. Try this. Open a "Command Prompt" window, cd to the folder where the .in file is located, and type
```
cfast jobname.in
```
where "jobname" is replaced by the name of your case.

**Reply by Antonella-unige** [reply]
Grazie Kevin, ci provo :)
Secondo te, posso provarci anche se al momento CFAST non è installato sul computer o se, anche se installato, è come se CFAST si fosse "corrotto" e non si apre più del tutto?

**Reply by drjfloyd** [reply]
Can you still run CEDIT (the graphical user interface), or is that not working either?

If you can run CEDIT, and the issue is you just cannot run or load your input file, then possibly some issue with the file.  If you can run CEDIT, copy the simple.in input from the CFAST install folder (C:\Program Files\firemodels\cfast7\Examples) to non-system folder and see if you can open and run that file using CEDIT. If that works, then the issue is probably the input you made.

**Reply by Antonella-unige** [reply]
Grazie Jason per il tuo suggerimento :)
Purtroppo, da quando mi è comparsa la schermata di errore che vi ho fatto vedere, CFAST/CEDIT proprio non si apre più e neppure permette di re-installarsi in modo da funzionare.
Il file .in sul quale stavo lavorando era salvato direttamente sul desktop del computer aziendale, quindi non era già in una cartella di sistema.
Se vi venissero altre idee per risolvere il problema, ve ne sarei molto grata.

**Reply by drjfloyd** [reply]
Did you uninstall CFAST or is the prior install still there? If it is still there, open a command window and type cfast. Do you see this:

CFAST

Release Version  : CFAST 7.7.3
Revision         : CFAST7.7.3-4-g0c33733b
Revision Date    : Thu May 12 10:19:24 2022 -0400
Compilation Date : Fri 05/13/2022  01:48 PM

Normal exit from CFAST

If this works, open your file with a text editor like notepad. Does it look like a text file or do you see some text and some garbage.  If you see garbage then something went wrong writing the file, If it looks like a normal text file, then change directories in your command window to where the file is and type (jobname is whatever your file name is):

cfast jobname.in

Does it run or do you see error messages? 

If it runs, then you would have the option of making your CFAST input by hand. It is just a text file. Not as convenient as the GUI.

Another option if you didn't unistall is open the control panel, go to remove programs, and uninstall cfast. That might let you reinstall.

**Reply by Antonella-unige** [reply]
Ci provo, grazie, e ti faccio sapere :)
